roles/platform/templates/mongo-connection-string.j2

```diff
@@ -9,7 +9,7 @@
     {% set tlsOptions = "&tls=true&tlsCAFile=" + itential_root_ca_file_destination %}
   {% endif %}
   {% if mongodb_auth | bool %}
-    {% set userpassword = "itential:" + mongo_user_itential_password + "@" %}
+    {% set userpassword = "itential:" + ( mongo_user_itential_password | urlencode | regex_replace("/","%2F") ) + "@" %}
```

Comment:
For reasons I can not explain the urlencode jinja filter will not escape the `/` character. Thus, it is escaped separately.